grails-app/i18n/messages.properties

```diff
@@ -806,6 +806,8 @@ register.confirm.success = Your account was confirmed successfully\!
 register.password.stablish = Establish now the password you will use to access your account.
 register.password.label = Establish your password
 register.password.repeat = Confirm your password
+campaign.closed.token.error.title = <span class="closedCampaignErrorTitle">Campaign Closed</span>
```

Comment:
Try to not use html code on properties. 
You can use if there aren't other way, like the languge change the order of the sentence or something like that. I think that in this case, the span should be in the gsp. 

OTHER POINT -> Add them to POE

Reply:
Done